src/platform/plugins/shared/dashboard/public/dashboard_api/panels_manager.ts

```diff
@@ -380,26 +396,28 @@ export function initializePanelsManager(
       },
       getState: (): {
         panels: DashboardState['panels'];
+        sections: DashboardState['sections'];
         references: Reference[];
       } => {
         const references: Reference[] = [];
 
         const panels = Object.keys(panels$.value).reduce((acc, id) => {
           const childApi = children$.value[id];
+          const panel = panels$.value[id];
           const serializeResult = apiHasSerializableState(childApi)
             ? childApi.serializeState()
             : getDashboardBackupService().getSerializedPanelBackup(id, dashboardId) ?? {
-                rawState: panels$.value[id].explicitInput ?? {},
+                rawState: panel.explicitInput ?? {},
                 references: getReferencesForPanelId(id),
               };
-          acc[id] = { ...panels$.value[id], explicitInput: { ...serializeResult.rawState, id } };
+          acc[id] = { ...panel, explicitInput: { ...serializeResult.rawState, id } };
 
           references.push(...prefixReferencesFromPanel(id, serializeResult.references ?? []));
 
           return acc;
         }, {} as DashboardPanelMap);
 
-        return { panels, references };
+        return { panels, sections: sectionsManager.api.sections$.getValue(), references };
```

Comment:
Why serialize sections here? How about just getting sections state in `get_dashboard_api.getState` call.

```
function getState() {
    const { panels, references: panelReferences } = panelsManager.internalApi.getState();
    const { state: unifiedSearchState, references: searchSourceReferences } =
      unifiedSearchManager.internalApi.getState();
    const dashboardState: DashboardState = {
      ...settingsManager.internalApi.getState(),
      ...unifiedSearchState,
      panels,
      sections: sectionsManager.internalApi.getState(),
      viewMode: viewModeManager.api.viewMode$.value,
    };
```

Reply:
Oops!! This was leftover from when I didn't have `sections_manager` as its own file and was doing everything in `panels_manager` 🙈 

Cleaned up in bed73385a862b5d06093ff4a5bc527cd311a61ae. Thanks for catching that!